3.0/s2i/bin/assemble

```diff
@@ -8,8 +8,7 @@ function rake_assets_precompile() {
   ! bundle exec 'rake -T' | grep "assets:precompile" >/dev/null && return
 
   echo "---> Starting asset compilation ..."
-  bundle exec "rake assets:precompile"
-}
```

Comment:
This probably should stay, otherwise the builds just fail when they try to invoke this.